chromium_src/chrome/browser/extensions/component_extensions_whitelist/whitelist.cc

```diff
@@ -11,17 +11,27 @@
 #include "brave/common/extensions/extension_constants.h"
 #include "brave/components/brave_extension/grit/brave_extension.h"
 #include "components/grit/brave_components_resources.h"
+#include "brave/components/brave_rewards/browser/buildflags/buildflags.h"
+#include "brave/components/brave_sync/buildflags/buildflags.h"
+#if BUILDFLAG(BRAVE_REWARDS_ENABLED)
 #include "brave/components/brave_rewards/resources/extension/grit/brave_rewards_extension_resources.h"
+#endif
+#if BUILDFLAG(ENABLE_BRAVE_SYNC)
 #include "brave/components/brave_sync/grit/brave_sync_resources.h"
+#endif
 #include "brave/components/brave_webtorrent/grit/brave_webtorrent_resources.h"
 
 namespace extensions {
 
   bool IsComponentExtensionWhitelisted(const std::string& extension_id) {
     const char* const kAllowed[] = {
       brave_extension_id,
+      #if BUILDFLAG(BRAVE_REWARDS_ENABLED)
```

Comment:
preprocessor directives should not be indented

Reply:
I have removed the indentation for preprocessor.